src/utils.js

```diff
@@ -5,21 +5,20 @@ const chalk = require('chalk');
 const constants = require('./constants');
 const CONFIG_FILE = constants.files.configFile;
 
-
-let error = (message) => {
+module.exports.error = (message) => {
     console.error(`neu: ${chalk.bgRed.black('ERROR')} ${message}`);
 }
 
-let isNeutralinojsProject = () => {
+module.exports.isNeutralinojsProject = () => {
     return fs.existsSync(CONFIG_FILE);
 }
 
-let getFiglet = () => {
+module.exports.getFiglet = () => {
     return figlet.textSync('Neutralinojs');
 }
 
 module.exports.figlet = () => {
-    console.log(getFiglet());
+    console.log(figlet.textSync('Neutralinojs'));
```

Comment:
Thanks for the updates..Now this code line seems duplicated I think. Also, I think the `checkCurrentProject` function will fail since `error(args)` is not defined. Do you think it's better to keep the source file in the original version? If we really need to make everything consistent in this file, as I understand, I think the only solution is to module.export all at the end of file, like this

```js
function a(){}
function b(){}
//...
//..
module.exports = {a, b};

```

Reply:
Umm. Few were exported directly and few were defined and exported. So, I just thought to change it to one format. You please let me know what to do. This is just a refactoring PR so anything works for me.

Reply:
@shalithasuranga what say, what do ? lets put it in one way rather than few direct exports and few implement + exports.

Reply:
@shalithasuranga ping

Reply:
Hey @pathange-s Thanks for reminding this again :) As I mentioned the current issue is the duplicated `textSync` call and [`error()`](https://github.com/neutralinojs/neutralinojs-cli/blob/908fca7b64ce60ee2e5ae3ff34209526cd9da1d3/src/utils.js#L26) function call (I will it will throw an error since the error function is not defined even though it's exported as `module.exports.error`).  

I think the original version is also okay. But, since you already submitted this change, I am suggesting you organize functions in a different way. The problem is that if we export a function with `module.exports.func`, we have to call it with either `module.exports.func()` or `this.func()` within the same file. Therefore I am suggesting you export all required functions at the end as I explained in the previous code snippet. Then, we can simply call `func()` within the same file and we can `require` them as usual.

Please let me know if you have questions or ideas. Thanks

Reply:
Gotcha ! Will do. Actually, defining all, then exporting it lgtm.

Reply:
Having function name as `figlet` and importing `figlet` (using require) is causing reference errors. 
[this](https://github.com/neutralinojs/neutralinojs-cli/blob/2611f36de9e94d47f6999aa1f0f86d295f19cfdb/src/utils.js#L3) and [this](https://github.com/neutralinojs/neutralinojs-cli/blob/2611f36de9e94d47f6999aa1f0f86d295f19cfdb/src/utils.js#L21)

Reply:
```
/home/sainath/gsoc/njs/neutralinojs-cli/src/utils.js:20
let figlet = () => {
    ^

SyntaxError: Identifier 'figlet' has already been declared
```

Reply:
Ah, maybe `showArt` ?

Reply:
@shalithasuranga updated it as required. Check it out.